.gitattributes

```diff
@@ -2,3 +2,41 @@
 # Set default behaviour, in case users don't have core.autocrlf set.
 * text=auto
 * text eol=lf
+
+# Explicitly declare text files we want to always be normalized and converted
+# to native line endings on checkout.
```

Comment:
Meh... it doesn't hurt leaving them around. It'll just be another thing to remember if any of these types of files start creeping into this repo at any point.